src/hw_simulator.js

```diff
@@ -110,4 +110,4 @@ module.exports = {
         platform.name = 'BeagleBone Simulator';
         return (platform);
     }
-}
+}
```

Comment:
Squash with beautify commit if this is a beautify setting. Why didn't it happen with my beautify?

Reply:
As i said before this was a mistake from my side , i did not have the beautify settings initially correct, i will squash the commits